### PR TITLE
Add desktop view to the web page

### PR DESCRIPTION
**Pull request by akhror-valiev**
- Used flexbox 
 - Used flex grid 
 - Added Desktop view
 - Added Media queries mobile responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -137,30 +152,37 @@ <h2>Multi-Post Stories</h2>
...
         </div>
+        <img
+          class="desktop"
+          src="images/Snapshoot Portfolio4.svg"
+          alt="Project photo"
+        />
       </div>
     </section>
 
     <section class="about-section">
```
Comment (nedjwab) [inline index.html:175]: [ ] kindly adjust the work section to be similar to the Figma design 👀, it should take 40% of the container's width, and 4 the text should be 4 to 5 lines.
Your design : 
histext

Figma design : 
destext

File: styles.css [hunk 2]
```diff
@@ -214,120 +221,120 @@ div a img {
...
-/* ----------------ABOUT SECTION---------------- */
+/* ==========================About Section ======================== */
 
 .about-section {
-  padding: 110px 24px;
-  display: flex;
-  flex-direction: column;
   background: white;
   border-top-right-radius: 100px;
+  display: grid;
```
Comment (nedjwab) [inline styles.css:229]: Nice using grid here 🔥🔥🔥🔥

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @akhror-valiev 

Nice job✔, I liked your implementation 🔥, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.
- Good design 🔥🔥🔥 , keep it up.😉

## (Changes Required) Aspects to improve: ♻️
You did great implementing  the project requirements 🔥🔥, but you still have some issues:  
Check the comments below 👇
  
## [Optional] suggestions:

-  Nothing to mention. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: styles.css (hunk 2 above)
Comment (akhror-valiev) [reply styles.css:229]: Thanks for your feedback @nedjwab!

File: styles.css [hunk 3]
```diff
@@ -214,120 +221,124 @@ div a img {
...
   background-repeat: no-repeat;
   background-size: 40vw;
   right: 80px;
   background-color: #6070ff;
-  margin: 114px 24px 0 24px;
+  margin-top: -10%;
   padding: 114px 24px;
   border-top-left-radius: 100px;
   background-position-x: right;
 }
```
Comment (Felix45) [inline styles.css]: [Optional] Please consider changing the `background-image`  property's value to `images/contact_form_bg.svg` , currently the background-image for this container is not displaying.  You can also consider making the background image for this section appear the way it is on the Figma template. Please see the screenshots below

```CSS
.contact-container {
  background-image: url(images/contact_form_bg.svg);
  background-repeat: no-repeat;
  background-size: 40vw;
  right: 80px;
  background-color: #6070ff;
  margin: 114px 24px 0 24px;
  margin-top: -10%;
  padding: 114px 24px;
  border-top-left-radius: 100px;
  background-position-x: right;
}
```

## Figma template


## Your Implementation

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @akhror-valiev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

## Highlights :heavy_check_mark: 
- [ ] Passing linter checks :clap: 
- [ ] Professional readme file :clap: 
- [ ] Descriptive pull request title & summary :clap
- [ ] Correct use of Gitflow :clap: 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Please the inline comment

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by akhror-valiev** [comment]
Thanks, @Felix45  it is noted and fixed.

**Pull request merged**